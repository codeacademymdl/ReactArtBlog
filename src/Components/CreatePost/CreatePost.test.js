import React from 'react';

import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import CreatePost from './CreatePost'

describe('CreatePost', () => {
    it('renders correctly', () => {
        const tree = renderer
          .create(<CreatePost/>)
          .toJSON();
        expect(tree).toMatchSnapshot();
      });
});

