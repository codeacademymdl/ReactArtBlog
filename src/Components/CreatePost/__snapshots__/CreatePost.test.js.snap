// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatePost renders correctly 1`] = `
<div
  className="createPost"
>
  <button
    className="button"
  >
    CreatePost
  </button>
</div>
`;
